guides/first_event_receiver.adoc

```diff
@@ -0,0 +1,22 @@
+= First Event Receiver
+
+:excerpt: This short guide will show you how to set up a Event receiver
+:title: First Event Receiver
+:order: 5
+:source-highlighter: highlightjs
+
+To write your first event receiver you write a component that implements the link:https://github.com/kreait/slack-spring-boot-starter/blob/master/starter/slack-spring-boot/src/main/kotlin/io/olaph/slack/broker/receiver/EventReceiver.kt[EventReceiver] interface you can use the  link:client/slack-api-client/src/main/kotlin/io/olaph/slack/client/SlackClient.kt[SlackClient] that is automatically registered with the starter
```

Comment:
I would do "To write your first event receiver you will need to write [..]".
Maybe start a new sentence after "interface" e.g. "[..] Interface. You can use the [..]"